### ISSUE TITLE
Create a Twilio like, plugin for Graylog.

### ISSUE THREAD

**Issue by leftyb** [Mobicents/RestComm#650]
It will be a great idea to use RestComm, to send emails and/or sms, in when an alarm on Graylog is triggered.   
https://marketplace.graylog.org/addons?search=twilio

**Comment by FerUy** [comment]
Agreed... but from experience, it can turn into a nightmare if you don't set thresholds like amount of SMS/emails per time unit and event.
